README.md

```diff
@@ -108,6 +108,19 @@ regexp=(https://localhost:9443/console|https://localhost:9443/t/(.*)/console|htt
 regexp=(https://localhost:9443/myaccount|https://localhost:9443/t/(.*)/myaccount|https://localhost:9443/myaccount/login|https://localhost:9443/t/(.*)/myaccount/login|https://localhost:9000/myaccount|https://localhost:9000/t/(.*)/myaccount|https://localhost:9000/myaccount/login|https://localhost:9000/t/(.*)/myaccount/login)
 ```
 
+---
+
+> [!IMPORTANT]
+> In Identity Server v7.0 and up, you need to configure the `callback_url` form the `deployment.toml` file.
+
+```toml
```

Comment:
Shall we add these steps under `### Setup WSO2 Identity Server` so that the dev/user can update deployment.toml in one go? In the current way it's written, a dev following the readme will have to shut down the server, add the callback URL configs and restart the server.

Just a suggestion :)